File: cpp2/progrm189.c



#include<stdio.h>

void DisplayR(char *str)
{
   while(*str!='\0')
   {
	   printf("%c\n",*str);
	   str++;
   }
}

int main()
{
    int iValue=0;
	char arr[20];
	printf("Enter the string\n");
	scanf("%[^'\n']s",&arr);
	DisplayR(iValue);	
	
	return 0;
}

File: cpp2/progrm189.java
import java.lang.*;
import java.util.*;


class progrm189
{
	public static void main(String argf[])
	{
		Scanner sobhj=new Scanner(System.in);
		System.out.println("Enter row");
		int irow=sobhj.nextInt();
		
		System.out.println("Enter col");
		int icol=sobhj.nextInt();
		
		int Arr[][]=new int[irow][icol];
		
		System.out.println("entter thje data");
		for(int i=0;i<Arr.length;i++)
		{
			for(int j=0;j<Arr[i].length;j++)
			{
				Arr[i][j]=sobhj.nextInt();
			}
		}
			System.out.println("elements are");
		for(int i=0;i<Arr.length;i++)
		{
			for(int j=0;j<Arr[i].length;j++)
			{
				System.out.print(Arr[i][j]+" ");
			}
			System.out.println(" ");
		}
		
		Arr=null;
		System.gc();
	}
}